libc/test/src/stdlib/StrfromTest.h

```diff
@@ -68,11 +69,27 @@ class StrfromTest : public LIBC_NAMESPACE::testing::Test {
     written = func(buff, 37, "A simple string with no conversions.", 1.0);
     ASSERT_STREQ_LEN(written, buff, "A simple string with no conversions.");
 
-    written =
-        func(buff, 37,
-             "%A simple string with one conversion, should overwrite.", 1.0);
-    ASSERT_STREQ_LEN(written, buff, is_long_double ? "0X8P-3" : "0X1P+0");
-
+    if (is_long_double) {
+      // 64-bit long double values require checking as they will give
+      // a different result
+#if defined(LIBC_TYPES_LONG_DOUBLE_IS_FLOAT64)
```

Comment:
`long double` can be 64, 80 or 128 bits so technically we'd need the following cases:
 - `LIBC_TYPES_LONG_DOUBLE_IS_FLOAT64`
 - `LIBC_TYPES_LONG_DOUBLE_IS_X86_FLOAT80`
 - `LIBC_TYPES_LONG_DOUBLE_IS_FLOAT128`

Reply:
@gchatelet is it alright if I simply add a test for 80-bit floating points, and add 128 bit tests later when I am more confident? I am not sure how to test the output for 128 bit floating point, and I am unable to find any resources currently, so I might need to do it manually.
Thanks!

Reply:
@gchatelet The tests work for 64 bit and 128-bit long doubles, but I am unsure on how to test for 80 bit ones. Do you have any pointers?

Reply:
@gchatelet I am sorry, I ran some tests on my computer using `libquadmath`, and I found that 64-bit and 128-bit floats give identical outputs, while 80-bits do not. I will update the tests, and re-push.
Thanks for the review!